components/menu/menu-item-styles.js

```diff
@@ -19,6 +19,8 @@ export const menuItemStyles = css`
 	:host(:hover),
 	:host(.focus-visible),
 	:host(.focus-visible[first]),
+	:host(:focus-visible),
+	:host(:focus-visible[first]),
```

Comment:
I confirmed that the new selectors work as expected in Chrome. I think we probably do want to add them separately instead of replacing the class selectors, since these styles could be applied to other components and I don't particularly want to break them if they opt to use the polyfill.

Reply:
I don't quite follow... why would other components put the `.focus-visible` CSS class on the menu item?

Reply:
Not a clue! If you don't think it's a risk we care about, I can remove it. I just opted for the least assumptions.

Reply:
Seeing as George added it as part of that [original change](https://github.com/BrightspaceUI/core/pull/1888/), I think it's pretty safe to remove.

Reply:
Fair, I guess it's only been ~2 months, so it's extremely unlikely anybody is doing anything weird with those directly. Mostly I'm usually concerned with "better safe than sorry" with things I'm likely going to hotfix so I'm more conservative than I need to be even with safe changes like this. 🤷